higlass_manage.py

```diff
@@ -411,8 +410,9 @@ def view(filename, hg_name, filetype, datatype, tracktype, position, public_data
 
     uid = json.loads(res.content)['uid']
 
-    webbrowser.open('http://localhost:{port}/app/?config={uid}'.format(
-        port=port, uid=uid))
+    if not os.environ.get('CI'):
```

Comment:
Maybe use a more obscure environment variable name
```suggestion
    # make sure this test passes on Travis CI and doesn't try to open
    # a terminal-based browser which doesn't return
    if not os.environ.get('HAS_JOSH_K_SEAL_OF_APPROVAL'):
```